pkg/microservice/aslan/core/environment/service/types.go

```diff
@@ -286,3 +290,41 @@ func (req *OpenAPIDeleteYamlServiceFromEnvReq) Validate() error {
 
 	return nil
 }
+
+type OpenAPIEnvCfgArgs struct {
+	Name             string                  `json:"name"`
+	EnvName          string                  `json:"env_name"`
+	ProductName      string                  `json:"product_name"`
+	ServiceName      string                  `json:"service_name"`
+	YamlData         string                  `json:"yaml_data"`
+	CommonEnvCfgType config.CommonEnvCfgType `json:"common_env_cfg_type"`
+	AutoSync         bool                    `json:"auto_sync"`
+}
+
+func (req *OpenAPIEnvCfgArgs) Validate() error {
+	_, err := commonrepo.NewEnvResourceColl().Find(&commonrepo.QueryEnvResourceOption{
```

Comment:
remove this, The validate function should only validate if the parameters are valid.